templates/skeleton/app/routes/cart.$lines.tsx

```diff
@@ -0,0 +1,69 @@
+import {redirect, type LoaderArgs} from '@shopify/remix-oxygen';
+
+/**
+ * Automatically creates a new cart based on the URL and redirects straight to checkout.
+ * Expected URL structure:
+ * ```ts
+ * /cart/<variant_id>:<quantity>
+ *
+ * ```
+ * More than one `<variant_id>:<quantity>` separated by a comma, can be supplied in the URL, for
+ * carts with more than one product variant.
+ *
+ * @param `?discount` an optional discount code to apply to the cart
+ * @example
+ * Example path creating a cart with two product variants, different quantities, and a discount code:
+ * ```ts
+ * /cart/41007289663544:1,41007289696312:2?discount=HYDROBOARD
+ *
+ * ```
+ * @preserve
```

Comment:
Can we remove this one as well? I don't think we need this docs in the final bundle 🤔